libs/cdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rx-angular/cdk",
-  "version": "1.0.0-alpha.11",
+  "version": "1.0.0-alpha.12",
```

Comment:
@hoebbelsB we should not update the lib manually on a feature branch, I revert this.